orca/orca.py

```diff
@@ -1919,7 +1920,9 @@ def run(steps, iter_vars=None, data_out=None, out_interval=1,
                     i, var))
 
         t1 = time.time()
-        for step_name in steps:
+        for j, step_name in enumerate(steps):
+            iter_step = namedtuple('iter_step', 'step_num,step_name')
```

Comment:
This doesn't need to be redefined in the loop, you can put it at the module level outside any function.